app/component/itinerary/StopCode.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const StopCode = props => <span className="itinerary-stop-code">{props.code}</span>;
```

Comment:
We also have a component called StopReference, should these somehow be consolidated? Or at least some kind of TODO added